### ISSUE TITLE
Word: Office.onReady being called twice in Word Online?

### ISSUE THREAD

**Issue by wandyezj** [OfficeDev/office-js#1121]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

## Expected Behavior

Office.onReady called once when Office is ready.

## Current Behavior

Office.onReady called twice

## Steps to Reproduce, or Live Example

https://word-twiddle.azurewebsites.net/manifest.xml

Load the manifest look in console.log that "Word Twiddle Ready!" is called twice.

## Context

I was counting on the logic in onReady to be called once, since that runs initialization code for my Add-In.

## Your Environment
* Platform Online
* Host Word
* Office version number: ______
* Operating System: ______
* Browser Chromium Edge

## Useful logs

![image](https://user-images.githubusercontent.com/4239645/80321265-8efb4280-87d0-11ea-841e-7377bcf85557.png)

![image](https://user-images.githubusercontent.com/4239645/80321248-77bc5500-87d0-11ea-82ce-555f249b56f4.png)

**Comment by MandytMSFT** [comment]
@wandyezj seems i can't access the manifest(browser says the resource not available) Would you please share it again?

**Comment by wandyezj** [comment]
@MandytMSFT I have fixed the manifest url. (although it's straight forward to reproduce the issue with any Word Add-In.
